components/features.tsx

```diff
@@ -1,322 +1,57 @@
-                Pear comes fully setup with shortcuts and terminal plugins used
-                to supercharge development productivity.
-              </p>
-            </div>
+function SingleFeature(props) {
 
-            {/* 6th item */}
-            <div
-              className="relative flex flex-col items-center"
-              data-aos="fade-up"
-              data-aos-delay="500"
-              data-aos-anchor="[data-aos-id-blocks]"
-            >
-              <svg
-                className="mb-4 h-16 w-16"
-                viewBox="0 0 64 64"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <rect
-                  className="fill-current text-primary-500"
-                  width="64"
-                  height="64"
-                  rx="32"
-                />
-                <g
-                  transform="translate(21 22)"
-                  strokeLinecap="square"
-                  strokeWidth="2"
-                  fill="none"
-                  fillRule="evenodd"
-                >
-                  <path
-                    className="stroke-current text-white-main"
-                    d="M17 2V0M19.121 2.879l1.415-1.415M20 5h2M19.121 7.121l1.415 1.415M17 8v2M14.879 7.121l-1.415 1.415M14 5h-2M14.879 2.879l-1.415-1.415"
-                  />
-                  <circle
-                    className="stroke-current text-white-main"
-                    cx="17"
-                    cy="5"
-                    r="3"
-                  />
-                  <path
-                    className="stroke-current text-white-main"
-                    d="M8.86 1.18C3.8 1.988 0 5.6 0 10c0 5 4.9 9 11 9a10.55 10.55 0 003.1-.4L20 21l-.6-5.2a9.125 9.125 0 001.991-2.948"
-                  />
-                </g>
-              </svg>
-              <h4 className="h4 mb-2">Familiar Feel</h4>
-              <p className="text-center text-lg text-gray-600">
-                Pear is a fork of VSCode, so you can pick up exactly where you
-                left off.
-              </p>
-            </div>
+  return (
+      <div className="flex flex-col m-24 items-center lg:flex-row lg:max-w-full lg:justify-around">
+          <div className="flex flex-col justify-center content-evenly text-center lg:text-left mb-8 lg:mb-0">
+              <h1 className="text-xl lg:text-4xl" style={{ color:"#4CC9A7", fontWeight:"500", whiteSpace: "pre-line" }} data-aos="fade-up" data-aos-delay="600">{props.info.header}</h1>
+              <br/>
+              <div className="text-base" style={{ whiteSpace: "pre-line" }} data-aos="fade-up" data-aos-delay="600"> {props.info.description} </div>
+          </div>
+          <div className=" lg:w-[870px] md:w-[600px] :w-[325px]">
+              <AspectRatio ratio={3 / 2}>
+                  <Image src={props.info.imageSource} alt="Image" className="rounded-lg object-cover" fill data-aos="fade-up" data-aos-delay="600"/>
```

Comment:
Also since this is a animated image/gif nextjs will try to optimize this and throw a error. Avoid this by adding the unoptimized property to the image component.